financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -124,9 +124,8 @@ private fun SuccessContentInternal(
             .padding(24.dp),
     ) {
         SpinnerToSuccessAnimation(
-            customMessage = payload?.customSuccessContent?.message,
-            customHeading = payload?.customSuccessContent?.heading,
-            accountsCount = payload?.accountsCount ?: 0,
+            content = payload?.content ?: TextResource.Text(""),
+            title = payload?.title ?: TextResource.Text(""),
```

Comment:
Nit: I’d be ok with keeping these nullable instead of the `""`.

Reply:
Actually we should support the _loading_ state (payload == null) on the success screen instead, I'll do that on a follow-up PR